### DISCUSSION TITLE
Bepinex doesn't install anything

### DISCUSSION THREAD

**Discussion by DanyJYT** [BepInEx/BepInEx#646] [Help and Q&A]
When i try to open the game up it doesn't install anything, even if i stayed in game for about 10 minutes, the game is Combat Master

![image](https://github.com/BepInEx/BepInEx/assets/66178534/216dc808-df7a-475d-a83b-ef2bc0ea3141)
![image](https://github.com/BepInEx/BepInEx/assets/66178534/87fb44a4-0ae1-4d13-90a1-149c1b431666)

**Comment by andre-schmidt-94** [comment]
Same thing here

**Comment by sine-of-theia** [comment]
having the same issue for Ultrakill, no clue whats going on
